lampo-common/src/model.rs

```diff
@@ -17,6 +17,7 @@ pub mod request {
     pub use crate::model::getinfo::*;
     pub use crate::model::invoice::request::*;
     pub use crate::model::keysend::request::*;
+    #[allow(unused_imports)]
```

Comment:
I think this is wrong, why we are using it? you are suppressing warning and we should not do without a good reason

Reply:
The network request has nothing in it. 
https://github.com/vincenzopalazzo/lampo.rs/blob/c10138add774f428b278224a4f3b19b4d5b24ff5/lampo-common/src/model/network.rs#L1

We could probably remove this, but then the request and response pattern would be non-uniform.